RocksdbMavenPomCache failure in tests

Building the maven plugin gives an error on my Windows machine. To my understanding that warning is okay and should not fail the test. This is very similar to #506.

## What is the full stack trace of any errors you encountered?
<!-- When errors occur, please include the output of `--stacktrace` for Gradle or `--debug` for Maven. -->
```
[ERROR] org.openrewrite.maven.RewriteCycloneDxIT.multi_module_with_independent_modules(MavenExecutionResult) -- Time elapsed: 8.709 s <<< FAILURE!
java.lang.AssertionError:

Expecting empty but was: ["Unable to initialize RocksdbMavenPomCache, falling back to InMemoryMavenPomCache"]
        at org.openrewrite.maven.RewriteCycloneDxIT.multi_module_with_independent_modules(RewriteCycloneDxIT.java:81)
```

## Are you interested in [contributing a fix to OpenRewrite](https://github.com/openrewrite/.github/blob/main/CONTRIBUTING.md#contributing-fixes)?
<!-- Indicate if this is something you would like to work on, and how we can best support you in doing so. -->
Sure. Same fix as in #506.

--- a/src/test/java/org/openrewrite/maven/RewriteCycloneDxIT.java
+++ b/src/test/java/org/openrewrite/maven/RewriteCycloneDxIT.java
@@ -78,7 +78,12 @@ class RewriteCycloneDxIT {
                 .withFile("b-1.0-cyclonedx.xml")
                 .exists();
 
-        assertThat(result).out().warn().isEmpty();
+        // ignore Maven POM cache warning, it may appear on some systems
+        // https://github.com/openrewrite/rewrite-maven-plugin/issues/636
+        assertThat(result).out().warn()
+                .filteredOn(warn -> !"Unable to initialize RocksdbMavenPomCache, falling back to InMemoryMavenPomCache"
+                        .equals(warn))
+                .isEmpty();
     }
 
     @MavenTest

